voidcrew/code/game/machinery/computer/shuttle/auto_pilot.dm

```diff
@@ -19,22 +27,54 @@
 	if(port?.current_ship)
 		ship = port.current_ship
 		return TRUE
+/*
+/obj/machinery/computer/autopilot/attackby(obj/item/O, mob/user, params)
```

Comment:
Why did you comment this out? just remove it

Reply:
forgot to remove that, left over from replacing it with multitool_act